doc/internal/testing.rst

```diff
@@ -50,7 +50,7 @@ The pyglet test suite is based on the `pytest framework <http://pytest.org>`_.
 It is preferred to use a virtual environment to run the tests.
 For instructions to set up virtual environments see :doc:`virtualenv`.
 Make sure the virtual environment for the Python version you want to
-test is active. It is preferred to run the tests on 3.6+
+test is active. It is preferred to run the tests on 3.7+
```

Comment:
To future proof this, perhaps we should just say something like: 
"To make sure changes are compatible with all supported Python versions, tests should ideally be run on all versions".

Reply:
It's future-proofed using a central source of truth on minimum versions. See new changes.